### DISCUSSION TITLE
Budget

### DISCUSSION THREAD

**Discussion by Daihana12** [OCA/odoo-community.org#151] [Q&A]
Hello!
Is it possible to create a budget in Odoo with three columns and modify the ones that appear by default?
Could a quote also be created without having the products recorded?
Greetings,

**Comment by vdewulf** [comment]
Hello,
You can use Mis Builder Budget to build your budget and compare to actual accounting data:
https://github.com/OCA/mis-builder/tree/16.0/mis_builder_budget

About your second question: I don't think it's possible to create a quote without having a product on the sale order line.
For future questions, you might open different discussions (one per topic)?

Have a nice day,
Virginie
